### ISSUE TITLE
Update for Sopel 8 + `pyproject.toml` packaging style

### ISSUE THREAD

**Issue by dgw** [sopel-irc/sopel-cookiecutter#13]
When installing a package that still has `setup.py` and `setup.cfg`, like the projects this template generates, `pip` or `setuptools` (can't be bothered to track down which) complains that a `setup.py` installation is deprecated.

This template should be updated for Sopel 8 anyway, and that'll be a good time to switch it over to `pyproject.toml`, maybe with a `Makefile` to go along with it so the package can be built with `make` instead of `python3 -m build`.

**Comment by dgw** [comment]
Mostly done with #15, though it didn't add a `Makefile`. Leaving open until someone gets to that (@Exirel, are you interested?)

**Comment by Exirel** [comment]
Yeah, #15 does most of the necessary work I think. At least from the packaging side.

A Makefile template could be nice, after all I pretty much use the same on all my projects.

**Comment by Exirel** [comment]
Oh and maybe update the Readme of this repo:

> Plugin template for **Sopel 7** and higher. To use:
